Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -1276,6 +1276,11 @@ void WebsiteDataStore::setResourceLoadStatisticsTimeAdvanceForTesting(Seconds ti
     protectedNetworkProcess()->setResourceLoadStatisticsTimeAdvanceForTesting(m_sessionID, time, WTFMove(completionHandler));
 }
 
+void WebsiteDataStore::setStorageAccessPromptQuirkForTesting(String topSite, Vector<String> subSites, CompletionHandler<void()>&& completionHandler)
```

Comment:
Nit - not too critical since this is just testing code, but we usually pass around `Vector<String>&&` or `const Vector<String>&` to avoid unnecessary copying.